src/fn_strings.cpp

```diff
@@ -102,10 +102,10 @@ namespace Sass {
       try {
         String_Constant* s = ARG("$string", String_Constant);
         str = s->value();
-        str = unquote(str);
+        // str = unquote(str);
```

Comment:
Consider removing these instead of commenting them out.

Reply:
I don't know yet if we can or can't remove those ... WIP ...

Reply:
Ah, sorry, missed the WIP tag